### ISSUE TITLE
Parsing error in transactsql - Show stopper

### ISSUE THREAD

**Issue by labs20** [taozhi8833998/node-sql-parser#1102]
**Describe the bug**
Trying to parse the follow sql:
```
select
trpriv_seq, trpriv_titulo, trpriv_id, trprivc_data
from termos_privacidade
left join termos_privacidade_versoes on (trprivv_trpriv_id = trpriv_id and trprivv_unidg_id is null and trprivv_inicio <= '2022-08-16T15:00:04.832Z' and (trprivv_fim >= '2022-08-16T15:00:04.832Z' or trprivv_fim is null))
left join termos_privacidade_consentimentos on (trprivc_trprivv_id = trprivv_id and trpriv_individual = 0 and trprivc_pes_id = 'null')
where 1 = 1 AND trprivv_id is not null   AND 1 = 2
order by 1,2
```
Renders this from in ast object
```
[
  {
    "db": null,
    "table": "termos_privacidade",
    "as": "left"       <<< ERROR HERE!!! This should be the first join type, not the alias for the FROM table
  },
  {
    "db": null,
    "table": "termos_privacidade_versoes",
    "as": null,
    "join": "INNER JOIN",       <<<< ERROR HERE!!! Now the first join has became INNER by default
    "on": {
      "type": "binary_expr",
      "operator": "AND",
      "left": {
        "type": "binary_expr",
        "operator": "AND",
        "left": {
          "type": "binary_expr",
          "operator": "AND",
          "left": {
            "type": "binary_expr",
            "operator": "=",
            "left": {
              "type": "column_ref",
              "table": null,
              "column": "trprivv_trpriv_id"
            },
            "right": {
              "type": "column_ref",
              "table": null,
              "column": "trpriv_id"
            }
          },
          "right": {
            "type": "binary_expr",
            "operator": "IS",
            "left": {
              "type": "column_ref",
              "table": null,
              "column": "trprivv_unidg_id"
            },
            "right": {
              "type": "null",
              "value": null
            }
          }
        },
        "right": {
          "type": "binary_expr",
          "operator": "<=",
          "left": {
            "type": "column_ref",
            "table": null,
            "column": "trprivv_inicio"
          },
          "right": {
            "type": "string",
            "value": "2022-08-16T15:00:04.832Z"
          }
        }
      },
      "right": {
        "type": "binary_expr",
        "operator": "OR",
        "left": {
          "type": "binary_expr",
          "operator": ">=",
          "left": {
            "type": "column_ref",
            "table": null,
            "column": "trprivv_fim"
          },
          "right": {
            "type": "string",
            "value": "2022-08-16T15:00:04.832Z"
          }
        },
        "right": {
          "type": "binary_expr",
          "operator": "IS",
          "left": {
            "type": "column_ref",
            "table": null,
            "column": "trprivv_fim"
          },
          "right": {
            "type": "null",
            "value": null
          }
        },
        "parentheses": true
      },
      "parentheses": true
    }
  },
  {
    "db": null,
    "table": "termos_privacidade_consentimentos",
    "as": null,
    "join": "LEFT JOIN",
    "on": {
      "type": "binary_expr",
      "operator": "AND",
      "left": {
        "type": "binary_expr",
        "operator": "AND",
        "left": {
          "type": "binary_expr",
          "operator": "=",
          "left": {
            "type": "column_ref",
            "table": null,
            "column": "trprivc_trprivv_id"
          },
          "right": {
            "type": "column_ref",
            "table": null,
            "column": "trprivv_id"
          }
        },
        "right": {
          "type": "binary_expr",
          "operator": "=",
          "left": {
            "type": "column_ref",
            "table": null,
            "column": "trpriv_individual"
          },
          "right": {
            "type": "number",
            "value": 0
          }
        }
      },
      "right": {
        "type": "binary_expr",
        "operator": "=",
        "left": {
          "type": "column_ref",
          "table": null,
          "column": "trprivc_pes_id"
        },
        "right": {
          "type": "string",
          "value": "null"
        }
      },
      "parentheses": true
    }
  }
]
```

**Database Engine**
SQLServer  (transactsql in options)

**Additional context**
As you can see, the 'left' from the first LEFT JOIN has became the alias for the table in FROM.

**Comment by labs20** [comment]
Moved back to 4.1.1 and the error is gone.

**Comment by taozhi8833998** [comment]
@labs20 fixed in the `4.5.0` version, give it a try.
